### ISSUE TITLE
key signatures: written vs. gestural

### ISSUE THREAD

**Issue by rettinghaus** [music-encoding/music-encoding#590]
Perhaps I missed something, but I don't understand why 
`scoreDef@key.accid` is of `data.ACCIDENTAL.GESTURAL`
whereas `keySig@accid` is of `data.ACCIDENTAL.WRITTEN`.

**Comment by lpugin** [comment]
@rettinghaus how changing this would break compatibility? In other words, 4.1 or 5.0?

**Comment by rettinghaus** [comment]
I don't think this would break anything in existing encodings. So more like `4.0.2`?

**Comment by bwbohl** [comment]
I think it is not really a mistake. When comparing the two data types, it becomes obvious that `data.ACCIDENTAL.GESTURAL` eliminates graphically distinct but logically identical values. The `<scoreDef>` as a basic _analytical_ definition of the score uses the gestural set for that reason. Whereas `<keySig>`(when occurring in `<layer>`) is part of the graphical domain. I understand that this somehow seems a bit awkward due to the name of the data type.

@rettinghaus is correct to assume changing the allowed values in `scoreDef@key.accid` to `data.ACCIDENTAL.WRITTEN` would not cause backward compatibility breaks.

This seems to bring up the whole domain discussions again (never mind, that's something good ;-) and the question would be what domain `<keySig>` in `<staffDef>` belongs to?!

**Comment by bwbohl** [comment]
maybe this could be tackled on the way with #1010

**Comment by bwbohl** [comment]
how is this related to #1170?

**Comment by musicEnfanthen** [comment]
I guess the change in #1170 made this issue obsolete. Do you agree @rettinghaus?

**Comment by rettinghaus** [comment]
@musicEnfanthen on the contrary. And @bwbohl's assumption is wrong. `scoreDef@key.accid` and `keySig@accid` are both in the analytical domain, because they both are additional indicators for the tonic, there is nothing visual about this. 

I'll prepare a PR to resolve this.

**Comment by rettinghaus** [comment]
@riedde why was this moved to Backlog?
